appConfig/appConfig.snd4.yaml

```diff
@@ -2,4 +2,10 @@
   value: "{{servicename}}-APPINSIGHTS-CONNECTIONSTRING"
   type: "keyvault"
 - key: GATEWAY_HOST
-  value: "https://ffc-sfd.snd4.adp.defra.gov.uk"
\ No newline at end of file
+  value: "https://ffc-sfd.snd4.adp.defra.gov.uk"
+- key: MESSAGE_HOST
+  value: "SNDADPINFSB4401.servicebus.windows.net"
+- key: FILE_RECEIVER_TOPIC_ADDRESS
+  value: "fcp-sfd-file-metadata"
+- key: FILE_RECEIVER_SUBSCRIPTION_ADDRESS
+  value: "fcp-sfd-file-receiver"
```

Comment:
We have to use `fcp` instead of `ffc` as the namespace prefix where all our microservices are hosted is called `fcp-sfd`.